2.14.9 release notes (#11294)
PrefectHQ/prefect: RELEASE-NOTES.md

# Prefect Release Notes

## Release 2.14.9

### Automatic infrastructure provisioning for ECS push work pools

> Following the introduction of [automatic project configuration for Cloud Run push pools](https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#automatic-project-configuration-for-cloud-run-push-work-pools) last week, we've added automatic provision infrastructure in your AWS and set up your Prefect workspace to support a new ECS push pool!

Reviewer: ```suggestion
Following the introduction of [automatic project configuration for Cloud Run push pools](https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#automatic-project-configuration-for-cloud-run-push-work-pools) last week, we've added the ability to automatically provision infrastructure in your AWS account and set up your Prefect workspace to support a new ECS push pool!
```

│          - Creating an IAM user for managing ECS tasks: prefect-ecs-user                                          │
│          - Creating and attaching an IAM policy for managing ECS tasks: prefect-ecs-policy                        │
│          - Storing generated AWS credentials in a block                                                           │
│          - Creating an ECS cluster for running Prefect flows: prefect-ecs-cluster                                 │
│          - Creating a VPC with CIDR 172.31.0.0/16 for running ECS tasks: prefect-ecs-vpc                          │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Proceed with infrastructure provisioning? [y/n]: y
Provisioning IAM user
Creating IAM policy
Generating AWS credentials
Creating AWS credentials block
Provisioning ECS cluster
Provisioning VPC
Creating internet gateway
Setting up subnets
Setting up security group
Provisioning Infrastructure ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
Infrastructure successfully provisioned!
Created work pool 'my-pool'!
```

If you have yet to try using an ECS push pool, now is a great time!

> If you use Azure, don't fret; we will add support for Azure Container Instances push work pools in future releases!

Reviewer: ```suggestion
If you use Azure, don't fret; we will add support for Azure Container Instances push work pools in a future release!
```

See the following pull request for implementation details:
— https://github.com/PrefectHQ/prefect/pull/11267

### Enhancements

> - Make Flows list a scannable table and standardize list headers throughout the app — https://github.com/PrefectHQ/prefect/pull/11274

Reviewer 2: @brandonreid  Is this worth highlighting?

> - Make flows list on flows page a scannable table and standardize list headers throughout the app — https://github.com/PrefectHQ/prefect/pull/11274

Reviewer 2: `Flow page in the Prefect UI`

> - Make flows list on flows page a scannable table and standardize list headers throughout the app — https://github.com/PrefectHQ/prefect/pull/11274

Reviewer 2: ```suggestion
- Make flows list on Flows page in the Prefect UI a scannable table — https://github.com/PrefectHQ/prefect/pull/11274
```
there's probably a better way to say it

### Fixes

> - Fix serve script crashes due to process limiter — https://github.com/PrefectHQ/prefect/pull/11264

Reviewer 2: ```suggestion
- Fix `.serve` script crashes due to process limiter — https://github.com/PrefectHQ/prefect/pull/11264
```
```suggestion
- Fix `.serve` crashes due to process limiter — https://github.com/PrefectHQ/prefect/pull/11264
```
